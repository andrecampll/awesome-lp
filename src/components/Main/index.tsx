const Main = () => (
  <div>
    <h1>Hello</h1>
  </div>
);

export default Main;
