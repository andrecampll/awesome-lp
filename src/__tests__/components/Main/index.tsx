import { render, screen } from '@testing-library/react';
import Main from '../../../components/Main';

describe('Main', () => {
  it('should be able to render', () => {
    const { container } = render(<Main />);

    expect(
      screen.getByRole('heading', {
        name: /Hello/i,
      }),
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
