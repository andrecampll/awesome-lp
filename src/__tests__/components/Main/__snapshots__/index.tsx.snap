// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should be able to render 1`] = `
<div>
  <h1>
    Hello
  </h1>
</div>
`;
